.github/actions/sccache/action.yml

```diff
@@ -89,31 +89,17 @@ runs:
         sessionToken: ${{ steps.aws-creds.outputs.aws-session-token }}
         use-fallback: false
 
-    - name: (External PR) Install AWS Signer (Linux)
-      if: >
-        runner.os == 'Linux' &&
-        github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name &&
-        runner.environment == 'self-hosted' &&
-        startsWith(runner.name, 'i-') != true
-      env:
-        LINUX_SIGNER_URL: https://rolesanywhere.amazonaws.com/releases/1.1.1/X86_64/Linux/aws_signing_helper
-      run: |
-        mkdir -p $HOME/.local/aws
-        curl $LINUX_SIGNER_URL -o $HOME/.local/aws/aws_signing_helper
-        chmod +x $HOME/.local/aws/aws_signing_helper
-      shell: bash
-
     - name: (External PR) Install AWS Signer (macOS)
       if: >
         runner.os == 'macOS' &&
         github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name &&
         runner.environment == 'self-hosted'
       env:
         DARWIN_SIGNER_URL: https://rolesanywhere.amazonaws.com/releases/1.1.1/X86_64/Darwin/aws_signing_helper
+        IAM_RA_PATH: ../../_aws
```

Comment:
I think we should avoid using relative paths. Let's just assume this tool is installed in a well known location.